src/frontend/Sider/Oppgavebenk/Oppgaverad.tsx

```diff
@@ -27,6 +27,10 @@ const Oppgaverad: React.FC<{ oppgave: Oppgave }> = ({ oppgave }) => {
         : behandlingstema;
     return (
         <Table.Row key={oppgave.id}>
+            <Table.DataCell>
+                {oppgave.oppgavetype && oppgaveTypeTilTekst[oppgave.oppgavetype]}
```

Comment:
```suggestion
                {oppgave.oppgavetype ? oppgaveTypeTilTekst[oppgave.oppgavetype]} : 'Mangler oppgavetype'
```

Nesten så vi burde ha en metode à la 
```typescript
const tekstEllerFeilmelding  = <TYPE extends string>(verdi: TYPE | undefined, record: Record<TYPE, string>) => {
    return verdi ? record[verdi] : `$verdi (Mangler mapping)`;
}
```